program_info/CMakeLists.txt

```
@@ -41,6 +41,21 @@ configure_file(org.prismlauncher.PrismLauncher.metainfo.xml.in org.prismlauncher
 configure_file(prismlauncher.rc.in prismlauncher.rc @ONLY)
 configure_file(prismlauncher.manifest.in prismlauncher.manifest @ONLY)
 configure_file(prismlauncher.ico prismlauncher.ico COPYONLY)
+
+if(MSVC)
+    set(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP true)
+    include(InstallRequiredSystemLibraries)
+    file(TO_NATIVE_PATH ${MSVC_REDIST_DIR} Native_MSVC_REDIST_DIR)
+    set(Launcher_MSVC_Redist_NSIS_Section "\
+Section \"Visual Studio Runtime\"
+  SetOutPath \"$INSTDIR\\vc_redist\"
+  File \"${Native_MSVC_REDIST_DIR}\\vc_redist.x64.exe\"
+  ExecWait \"$INSTDIR\\vc_redist\\vc_redist.x64.exe /install /passive\"
```

```suggestion
  ExecWait \"$INSTDIR\\vc_redist\\vc_redist.x64.exe /install /passive /norestart\"
```
Nitpick, but this is what's [recommended by Microsoft](https://learn.microsoft.com/en-us/cpp/windows/redistributing-visual-cpp-files?view=msvc-170#command-line-options-for-the-redistributable-packages)

> You can also specify a /quiet option, which doesn't display any UI or require any user interaction.

This also doesn't sound like a bad idea

---

when using the `/quiet` options and executed through NSIS there is no progress feedback. IMO I think the  `/passive`  path is best

---

Can't this trigger the "Modify Setup" dialog (like you linked [here](https://discord.com/channels/1031648380885147709/1031823065937629267/1289121821253439519)) when omitted though? If not, then yeah I'm fine without it

---

No, that was before i put the `/passive` option it. with `/passive` it shows gui but doesn't prompt the user beyond triggering UAC

---

Ah I see. Thanks!